src/style.css

```diff
@@ -58,14 +58,18 @@
 }
 
 .stars__item {
+  display: flex;
+  align-items: center;
+  background: url(/images/star.svg) no-repeat center;
+  background-size: auto;
   width: 15px;
   height: 15px;
   margin-left: 4px;
-  display: flex;
-  align-items: center;
 }
 
-.stars__container :last-child {
+.stars__container :nth-child(-n+4) {
```

Comment:
add rules for all cases(from 1 to 5 stars) and add modifier to class --4/--3 and so on